### DISCUSSION TITLE
Unable to install library

### DISCUSSION THREAD

**Discussion by AsadR110** [esp8266/Arduino#7778] [Q&A]
I think this question was already answered but I am not able to install the library using the link which was provided. This is the error I am getting 
`Error downloading https://arduino.esp8266.com/stable/package_esp8266com_index.json`
Is there a problem with the link. I would really appreciate some assistance.

**Comment by devyte** [comment]
The link above works, just type it into a browser and you'll see the json.

**Reply by AsadR110** [reply]
After opening it on my browser how can I install it on the arduino ide.

**Comment by imiaozi** [comment]
I cann't install too. And I found an issue in package_esp8266com_index.json.

"tools": [
        {
          "version": "3.7.2-post1",
          "name": "python3",
          "systems": [
            {
              "host": "x86_64-mingw32",
              "url": "https://github.com/earlephilhower/esp-quick-toolchain/releases/download/2.5.0-4/python3-3.7.2.post1-embed-win32v2a.zip", 

**### here is win32v2a.zip**

              "archiveFileName": "python3-3.7.2.post1-embed-win32v2a.zip",

**### here is win32v2a.zip**

              "checksum": "SHA-256:f57cb2daf86176d2929e7c58990c2ac32554e3219d454dcac10e464ddda35bf2",
              "size": "6428926"
            },
            {
              "host": "i686-mingw32",
              "url": "https://github.com/earlephilhower/esp-quick-toolchain/releases/download/2.5.0-4/python3-3.7.2.post1-embed-win32v2a.zip",

**### here is win32v2a.zip**

              "archiveFileName": "python3-3.7.2.post1-embed-win32va2.zip",

**### but here is win32v a 2.zip**

              "checksum": "SHA-256:f57cb2daf86176d2929e7c58990c2ac32554e3219d454dcac10e464ddda35bf2",
              "size": "6428926"
            },


Just now I sovled this problem by  copying a "win32v2a.zip" in "staging/packages" folder using a name "win32va2.zip"

**Comment by earlephilhower** [comment]
This should now be fixed in the 2.7.4 release (I added a file with the flubbed name) and #7847 will fix the 3.0.0 release going forward.  Thanks for reporting, @imiaozi .
